token/js/src/extensions/index.ts

```diff
@@ -3,6 +3,7 @@ export * from './cpiGuard/index.js';
 export * from './defaultAccountState/index.js';
 export * from './extensionType.js';
 export * from './groupPointer/index.js';
+export * from './groupMemberPointer/index.js';
```

Comment:
micro-nit: you did this alphabetically earlier in the PR, so maybe this should go one line above?

Reply:
RIP.